test/unit-test/dawn/Optimizer/TestPartitionAlgorithm.cpp

```diff
@@ -1,13 +1,13 @@
 //===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
filename here:
name the file to Pass<FILENAME> where <FILENAME> is the file where the function is present